src/rpc/blockchain.cpp

```diff
@@ -184,14 +174,22 @@ UniValue getbestblockhash(const JSONRPCRequest& request)
     return chainActive.Tip()->GetBlockHash().GetHex();
 }
 
-void RPCNotifyBlockChange(bool ibd, const CBlockIndex * pindex)
-{
-    if(pindex) {
-        std::lock_guard<std::mutex> lock(cs_blockchange);
-        latestblock.hash = pindex->GetBlockHash();
-        latestblock.height = pindex->nHeight;
-    }
-    cond_blockchange.notify_all();
+template<typename T>
+static UniValue AwaitBlockChangeCondition(int timeout, const T& term_condition) {
```

Comment:
style nit: "break before braces on function, namespace and class definitions"

https://github.com/bitcoin/bitcoin/blob/master/src/.clang-format#L15
http://clang.llvm.org/docs/ClangFormatStyleOptions.html

Reply:
Fixed (for at least not-two-line-functions).